### PR TITLE
Advent 2017 - gRPC Go : Beyond the basics (13 Dec)

### PR DESCRIPTION
**Pull request by pieterlouw**
This is my first draft of my post scheduled for 13 December - **gRPC Go: Beyond the basics**.

I'm open to any corrections on errors and wrong wording & suggestions.

Thanks for the opportunity.

### CODE REVIEW

File: content/advent-2017/go-grpc-beyond-basics.md [hunk 1]
```diff
@@ -0,0 +1,501 @@
++++
+author = ["Pieter Louw"]
+date = "2017-12-13"
+series = ["Advent 2017"]
+linktitle = "gRPC Go: Beyond the basics"
+title = "gRPC Go: Beyond the basics"
+
```
Comment (shurcooL) [inline content/advent-2017/go-grpc-beyond-basics.md]: I don't know whether it affects parsing or not, but you should remove this blank line (unless you have a good reason for including it). Other posts don't have it. See https://raw.githubusercontent.com/gopheracademy/gopheracademy-web/master/content/advent-2017/mphf.md for example.

**Comment by pieterlouw** [comment]
Hi @shurcooL ,
Thanks for that. There's no good reason to have it there.  I removed it now.

File: content/advent-2017/go-grpc-beyond-basics.md [hunk 2]
```diff
@@ -0,0 +1,500 @@
...
+There are many other options like message and buffer sizes and specifying a custom codec (something other than Protocol Buffer) available.To see what other options are available the API docs are your friend.
+
+For more server side options, see [ServerOption](https://godoc.org/google.golang.org/grpc#ServerOption).
+
+For more client side options, see [DialOption](https://godoc.org/google.golang.org/grpc#DialOption).
+
+**Tip:** `DialOption` values all have a **With** prefix, i.e grpc.**With**TransportCredentials
+
+*Fo more resources on enabling TLS for gRPC in Go:*
```
Comment (shurcooL) [inline content/advent-2017/go-grpc-beyond-basics.md]: "Fo" looks like a typo, should be "For".

Comment (pieterlouw) [reply content/advent-2017/go-grpc-beyond-basics.md]: Fixed, thanks.

**Pull request merged**